MetaMask/metamask-extension
# Fix: show whats new to users who created, not imported, a new wallet,…

… but not on their first session

Fixes a bug introduced in https://github.com/MetaMask/metamask-extension/pull/13886 that would prevent whats new messages from being shown to users who had completed their installs with the "Create a Wallet" option

## Explanation

Code was previously relying on `completedOnboarding` to decide whether to show the user what's new, when if fact we needed a variable like `completedOnboardingInThisUISession`. The goal was to not show the popup to entirely new users. The problem is that `completedOnboarding` is set to true once and stays true forever, so users were not seeing it in any session. This PR adds a new variable to redux state to decide whether to show whats new to users.

## Testing steps

1. Locally build on an old commit. v10.7.0 or something like
2. Create a new seed phrase while onboarding
3. Locally build on develop, reload the extension
4. Close the portfolio dapp tooltip
5. Refresh the page
6. Notice that there are no whats new popups (if you want, log in and out to still see no whats new popup)
7. Build on this PRs branch
8. Reload the extension
9. Log in and see the whats new popup
